### DISCUSSION TITLE
Where I can find changelog for old webpack-dev-server releases?

### DISCUSSION THREAD

**Discussion by matkoniecz** [webpack/webpack#18161] [Q&A] [answered]
I have a project using ancient webpack-dev-server (3.11.3) which I want to update.

Obviously, I should check changelogs for potential breaking changes to know what I should check.

I found https://github.com/webpack/webpack/releases?page=36 with for example https://github.com/webpack/webpack/releases/tag/v4.0.0-alpha.0 - but is it also release of webpack-dev-server ? Or is it having own releases/changelog?

What worries me is that I apparently use webpack-dev-server 3.11.3, version not listed at I found https://github.com/webpack/webpack/releases?page=36 between [v3.10.0](https://github.com/webpack/webpack/releases/tag/v3.10.0) and [v4.0.0-alpha.0](https://github.com/webpack/webpack/releases/tag/v4.0.0-alpha.0)

**Comment by alexander-akait** [comment, marked as answer]
https://github.com/webpack/webpack-dev-server/releases and https://github.com/webpack/webpack-dev-server/blob/master/migration-v4.md and https://github.com/webpack/webpack-dev-server/blob/master/migration-v5.md
